Dockerfile

```diff
@@ -42,7 +42,7 @@ WORKDIR /node/src/app
 COPY web .
 
 # Install the dependencies and build
-RUN npm ci && npm run build
+RUN yarn install && yarn build
```

Comment:
What is the rationale for moving from npm to yarn?

Reply:
Everything I've read suggested that yarn is faster than npm and that it's the natural successor of npm. Happy to change it back if we're overly concerned.

Reply:
it's fine, just wanted to confirm.